ariac_gazebo/ariac_gazebo/environment_startup.py

```diff
@@ -154,7 +167,7 @@ def spawn_kit_trays(self):
         for id, slot in zip(ids, slots):
             rel_pose = Pose()
             rel_pose.position.x = slot_info[slot]["offset"]
-            rel_pose.position.z = .01
+            rel_pose.position.z = .015
```

Comment:
Should this number be a parameter?

Reply:
Actually, this can be reverted to .01 and yeah it can be named to tray_thickness. I increased it when I was trying to see if I spawned the tray a little bit raised the contacts would show up but it didn't work.